winml/lib/Api.Image/inc/D3DDeviceCache.h

```diff
@@ -16,24 +16,11 @@
 
 namespace _winml {
 
-enum class PipelineStateCacheType : unsigned char {
-  kFloat32 = 0,
-  kFloat16 = 1,
-  kCount = 2
-};
+enum class PipelineStateCacheType : unsigned char { kFloat32 = 0, kFloat16 = 1, kCount = 2 };
```

Comment:
Is there a setting to keep enums on separate lines still (even if they would all fit under 120 columns)? I think separate lines are generally nicer to scan through the list than cramming them all together.

Reply:
I think so! Let me add that

Reply:
Done